src/psyclone/tests/test_files/dynamo0p3/infrastructure/argument_mod.F90

```diff
@@ -213,11 +213,7 @@ module argument_mod
 
   !> @defgroup operates_on Enumeration of kernel iterator property descriptors.
   !> @{
-  ! Note: `iterates_over = CELLS` changes to `operates_on = CELL_COLUMN` in
-  !       PSyclone 2.0.0. Currently both options are supported for uninterrupted
-  !       development, however the `CELLS` option will be removed in #870 or #874).
   integer, public, parameter :: CELL_COLUMN = 396
```

Comment:
I would have expected to see DOF in here but I've grep'd and it isn't. However, the built-in definitions file (https://github.com/stfc/PSyclone/blob/f33db525f522ef5cbb4263093b4706bb119fc1a9/src/psyclone/parse/lfric_builtins_mod.f90#L48-L53) needs it.

Reply:
That's strange, the built-ins test with compilation worked. I renamed the former `CELLS` id to `DOF`.

Reply:
We never compile this file - it's just used to provide the kernel metadata.